docs/sources/mimir/release-notes/v2.8.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Grafana Mimir version 2.8 release notes"
+menuTitle: "V2.8 release notes"
+description: "Release notes for Grafana Mimir version 2.8"
+weight: 300
+---
+
+# Grafana Mimir version 2.8 release notes
+
+Grafana Labs is excited to announce version 2.8 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Query-frontend and ruler now use protobuf internal query result payload format by default**.
```

Comment:
@charleskorn Can you help to add more color to this? What's the benefit?

Reply:
I'd add something like: "This reduces the CPU and memory utilisation of the querier, query-frontend and ruler, as well as reducing network bandwidth consumed between these components."